# Robot parameters 
tcp_speed: 0.05

# Gripper parameters
tcp2fingertip: 0.28 #0.275 # distance from tcp to gripper fingertip
opening_per_count: 0.00065 # gripper stroke opening per rPr count
finger_thickness: 0.0075
max_opening: 0.1523 # max stroke of gripper excluding finger thickness
gripper_speed: 0.1 # value between 0.013 and 0.100
gripper_force: 10 # value between 0 and 100
gripper_offset: 0.003 #gripper postion offset for compensating the error

# Object dimension
object_thickness: 0.014  # object thickness in meters
object_length: 0.049 # object length in meters

# Initial configuration
delta_0: 0.03 # distance from fingertip to object tip within gripper
theta_0: 45.0 #Angle between object and hole prior to regrasping

# Intermediate configuration
psi_regrasp: 24 #Angle between object and upper finger after regrasping
theta_tilt: 13 #Angle to tilt after regrasping
tuck: 22
tuck_dist: 0.005 #distance from fingertip to contact B before tuck

# Action axis
axis: [-1, 0, 0]

# Simulation
sim: 0